ios/Classes/NativeBridge/ModuleExtensions/OneginiModuleSwift+Auth.swift

```diff
@@ -76,91 +71,64 @@ extension OneginiModuleSwift {
             error != nil ? callback(SdkError.convertToFlutter(error)) : callback(String.stringify(json: result ?? []))
         })
     }
-    
-    func authenticateWithRegisteredAuthentication(_ identifierId: String?, callback: @escaping FlutterResult) {
-        guard let _identifierId = identifierId else {
-            callback(SdkError.convertToFlutter(SdkError(.authenticatorIdIsNull)))
-            return
+
+    func authenticateWithRegisteredAuthentication(_ profileId: String, _ registeredAuthenticatorId: String, callback: @escaping FlutterResult) {
+        guard let profile: ONGUserProfile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.profileId == profileId }) else {
```

Comment:
you don't needed that `: ONGUserProfile `

Reply:
done